test/Recipe-test.js

```diff
@@ -0,0 +1,113 @@
+import { expect } from 'chai';
+import Recipe from '../src/classes/Recipe';
+
+describe('Recipe', () => {
+    it('Should be a function', () => {
+      expect(Recipe).to.be.a('function');
+    });
+
+let recipe1;
```

Comment:
something funny is going on with indentation here. these let statements should be intended, because they are children of the describe block.

Reply:
Good eye, thanks!